scripts/lib/libvirt/cleanup.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import libvirt
+import sys
+import os
+import glob
+import subprocess
+import argparse
+
+conn = libvirt.open("qemu:///system")
+if not conn:
+    print "Failed to open connection to the hypervisor"
```

Comment:
Uhm. I would prefer to not to do that and use the print statement for Python 2 and py2to3 to takes care of that.  If the team decide to go for Python 3 now, for me makes sense.

Also is a bad idea use `print("")` without `from __future__ import print_function`